src/Microsoft.Health.Dicom.Core/Features/Operations/IDicomOperationsClient.cs

```diff
@@ -75,24 +91,13 @@ public interface IDicomOperationsClient
     /// <summary>
     /// Asynchronously begins the instance blob copy.
     /// </summary>
+    /// <param name="operationId">The desired ID for the copy operation.</param>
+    /// <param name="watermarkRange">Optional watermark range to use on interruption.</param>
     /// <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="CancellationToken.None"/>.</param>
     /// <returns>
     /// A task representing the <see cref="StartBlobCopyAsync"/> operation.
-    /// The value of its <see cref="Task{TResult}.Result"/> property contains the ID of the operation
-    /// that is performing the asynchronous addition.
-    /// </returns>
-    /// <exception cref="OperationCanceledException">The <paramref name="cancellationToken"/> was canceled.</exception>
-    Task<Guid> StartBlobCopyAsync(CancellationToken cancellationToken = default);
-
-    /// <summary>
-    /// Asynchronously gets the completed status of blob copy.
-    /// </summary>
-    /// <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="CancellationToken.None"/>.</param>
-    /// <returns>
-    /// A task representing the <see cref="IsBlobCopyCompletedAsync"/> operation.
-    /// The value of its <see cref="Task{TResult}.Result"/> property a value that indicate whether the copy
-    /// operation as completed.
+    /// The value of its <see cref="Task{TResult}.Result"/> that is performing the asynchronous addition.
     /// </returns>
     /// <exception cref="OperationCanceledException">The <paramref name="cancellationToken"/> was canceled.</exception>
-    Task<bool> IsBlobCopyCompletedAsync(CancellationToken cancellationToken = default);
+    Task StartBlobCopyAsync(Guid operationId, WatermarkRange? watermarkRange, CancellationToken cancellationToken = default);
```

Comment:
Perhaps give the `watermarkRange` a default value of `null`?

Reply:
Done